ethcore/private-tx/src/lib.rs

```diff
@@ -674,6 +689,11 @@ impl Provider {
 		Ok(result.result)
 	}
 
+	/// Retrieves log information about private transaction
+	pub fn private_log(&self, tx_hash: H256) -> Result<TransactionLog, Error> {
+		self.logging.tx_log(&tx_hash).ok_or(ErrorKind::TxNotFoundInLog.into())
```

Comment:
`ok_or` is eager (always evaluated) could be replaced by `ok_or_else` because it is lazy